pages/cards/cards.css

```diff
@@ -44,16 +55,6 @@
   padding: 10px;
   display: block;
 }
-.text-over-media {
-  position: relative;
-}
-.card-header {
-  position: absolute;
-  /* color: white; */
-  /* bottom: 0; */
```

Comment:
Try deleting these comments if not necessary.

Reply:
Those are deleted comments.
There are a few littered throughout the CSS of all components should I remove them all in one commit?